stack/README.md

```diff
@@ -15,46 +15,23 @@ inappropriate for the task at hand. This allows us to practice our value of
 [Continuous Improvement]: https://thoughtbot.com/purpose#continuous-improvement
 [Quality]: https://thoughtbot.com/purpose#quality
 
-## Layers
+## Core Stack
 
-Our stack is broken up into layers. Each layer depends on another layer. This
-allows us to swap out an entire layer when necessary without losing all the
-decisions made for other layers.
+Most developers at thoughtbot learn our Core Stack. The stack is broken up into
+layers and each layer depends on another layer. This allows us to swap out an
+entire layer when necessary without losing all the decisions made for other
+layers. Developers will have varying levels of experience with each layer, but
+the gaps between layers are small enough that developers can learn a new layer
+while building applications.
 
 ### UI
```

Comment:
Should this be "Web UI" instead? I don't expect e.g. Android UI to use server-rendered HTML.

Reply:
This is also attempting to say that an HTML-based web UI will be the default UI. If we don't need to build a mobile application, we won't.

This section and the server-rendered HTML guideline weren't added by this pull request, so I would open a new issue or pull request if you think it should change.